Gandalan.IDAS.WebApi.Client/BusinessRoutinen/AVBerechnungCloudRoutinen.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Gandalan.IDAS.Client.Contracts.Contracts;
+using Gandalan.IDAS.WebApi.Data.DTOs.Produktion;
+using System;
+
+namespace Gandalan.IDAS.WebApi.Client.BusinessRoutinen
+{
+    public class AVBerechnungCloudRoutinen : WebRoutinenBase
+    {
+        public AVBerechnungCloudRoutinen(IWebApiConfig settings) : base(settings)
+        {
+        }
+
+        public BerechnungResultDTO Process(string varianteName, BerechnungParameterDTO parameter)
```

Comment:
varianteName is contained in BerechnungParameterDTO.BelegPositionAVDTO.Variante, please use that instead of the parameter

Reply:
done